Automattic/woocommerce-payments
# Adding the enabled status and the logic to render the Chip component.

Fixes #5966 

#### Changes proposed in this Pull Request





Update to add compatibility for a server PR which returns the status as `Enabled` when there is a future requirement but no deadline. This will replace `Restricted Soon` in cases where there is no deadline for the user to take action.





#### Testing instructions





* Pull this branch and run `npm run watch`
* Ensure you have an account with no requirements (status should be `Complete`)
* Adjust the `get_basic_account_info` function on the server to return `enabled` as the `status` in the account response.
* Go to the `Payments > Overview` page, verify that the account status display looks as follows:

<img width="773" alt="Screenshot 2023-08-03 at 11 41 35" src="https://github.com/Automattic/woocommerce-payments/assets/11770181/bb16f885-dc7d-4091-bb02-221117cff373">

* Reverse the change you made to the account status response.
* Refresh the page and verify that the `Complete` status displays as follows (note that it has changed from blue to green, because enabled is blue matching how Stripe display it): 

<img width="724" alt="Screenshot 2023-08-03 at 11 41 53" src="https://github.com/Automattic/woocommerce-payments/assets/11770181/28598a29-6f67-4a35-84fd-c7dc9051bc54">


-------------------


**Post merge**
